packages/app-desktop/gui/MainScreen/MainScreen.tsx

```diff
@@ -156,6 +158,23 @@ class MainScreenComponent extends React.Component<Props, State> {
 		this.layoutModeListenerKeyDown = this.layoutModeListenerKeyDown.bind(this);
 
 		window.addEventListener('resize', this.window_resize);
+
+		ipcRenderer.on('asynchronous-message', (_event: any, message: string, args: any) => {
```

Comment:
"event"

Reply:
It is unused here, and so if I remove the underscore I get a warning from the linter